pkg/cmd/infra/router/router.go

```diff
@@ -37,35 +54,75 @@ type templateRouterConfig struct {
 	StatsUsername      string
 }
 
+// f5RouterConfig is the config necessary to start an F5 router plugin
+type f5RouterConfig struct {
```

Comment:
Will the f5 router not support using a wildcard certificate for auto-generated route names?  I see that `DefaultCertificate` as left in the `templateRouterConfig` but it seems like it should be relevant to both implementations.